### ISSUE TITLE
STASH to CF mapping: Use of aliases

### ISSUE THREAD

**Issue by tv3141** [SciTools/iris#2724]
The names below are aliases for CF standard names being used in the STASH to CF mapping. To avoid ambiguity I suggest to avoid aliases.

```bash
!wget http://cfconventions.org/Data/cf-standard-names/55/build/cf-standard-name-table.html
!wget https://raw.githubusercontent.com/SciTools/iris/master/lib/iris/fileformats/um_cf_map.py
```

```python
import um_cf_map

cf_file = 'cf-standard-name-table.html'

aliases = {}  # alias -> name
with open(cf_file) as f:
    lines = list(f)
    for i, line in enumerate(lines):
        if 'alias' in line:
            alias = line[14:-7]
            
            # go back to line with proper name
            while not 'name' in lines[i]:
                i -= 1
            name = lines[i][9:90].split('"')[0]
            
            aliases[alias] = name

iris_cf_names = [val.standard_name for val in um_cf_map.STASH_TO_CF.values()]
aliases_in_um_cf_map = set(aliases.keys()).intersection(set(iris_cf_names))
for alias in aliases_in_um_cf_map:
    print('{}\n    -> {}'.format(alias, aliases[alias]))
```
```
air_pressure_at_sea_level
    -> air_pressure_at_mean_sea_level
atmosphere_cloud_liquid_water_content
    -> atmosphere_mass_content_of_cloud_liquid_water
large_scale_snowfall_flux
    -> stratiform_snowfall_flux
soil_moisture_content
    -> mass_content_of_water_in_soil
mass_concentration_of_dust_dry_aerosol_in_air
    -> mass_concentration_of_dust_dry_aerosol_particles_in_air
atmosphere_cloud_ice_content
    -> atmosphere_mass_content_of_cloud_ice
gross_primary_productivity_of_carbon
    -> gross_primary_productivity_of_biomass_expressed_as_carbon
soil_respiration_carbon_flux
    -> surface_upward_mass_flux_of_carbon_dioxide_expressed_as_carbon_due_to_respiration
tendency_of_mass_fraction_of_stratiform_cloud_ice_in_air_due_to_riming_from_cloud_liquid
    -> tendency_of_mass_fraction_of_stratiform_cloud_ice_in_air_due_to_riming_from_cloud
net_primary_productivity_of_carbon
    -> net_primary_productivity_of_biomass_expressed_as_carbon
plant_respiration_carbon_flux
    -> surface_upward_mass_flux_of_carbon_dioxide_expressed_as_carbon_due_to_plant_respi
atmosphere_optical_thickness_due_to_seasalt_ambient_aerosol
    -> atmosphere_optical_thickness_due_to_sea_salt_ambient_aerosol_particles
surface_carbon_dioxide_mole_flux
    -> surface_upward_mole_flux_of_carbon_dioxide
large_scale_rainfall_flux
    -> stratiform_rainfall_flux
moisture_content_of_soil_layer
    -> mass_content_of_water_in_soil_layer
tendency_of_mass_fraction_of_stratiform_cloud_ice_in_air_due_to_heterogeneous_nucleation_from_cloud_liquid
    -> tendency_of_mass_fraction_of_stratiform_cloud_ice_in_air_due_to_heterogeneous_nuc
atmosphere_optical_thickness_due_to_dust_ambient_aerosol
    -> atmosphere_optical_thickness_due_to_dust_ambient_aerosol_particles
surface_snow_and_ice_sublimation_flux
    -> tendency_of_surface_snow_and_ice_amount_due_to_sublimation
soil_carbon_content
    -> soil_mass_content_of_carbon
```

**Comment by pelson** [comment]
Thanks @tv3141. I agree. There is some complexity here as we source our data from metaRelate. (see comment at https://github.com/SciTools/iris/blob/master/lib/iris/fileformats/um_cf_map.py#L18-L23).

Long term we need to make a decision as to whether having that data outside of the Iris codebase is healthy. I fear we aren't going to be able to make that decision quickly though - there are pros/cons on both sides.

**Comment by tv3141** [comment]
@marqh I saw you fixed #2725 in #3043. Is there any chance you can fix this issue too?
I updated the issue description to show the proper standard name for each alias that is being used.
